<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Degisrad</class>
 <widget class="QMainWindow" name="Degisrad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>151</width>
    <height>221</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DegToRad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Градусы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_deg">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maxLength">
       <number>1024</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Минуты</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_min">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maxLength">
       <number>1024</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Секунды</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_sec">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="maxLength">
       <number>1024</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_do">
      <property name="text">
       <string>Перевод</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Радианы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_rad">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>line_rad</tabstop>
  <tabstop>btn_do</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
